src/test/functional/cypress/e2e/integration/api/admin/empStatus_API.cy.js

```diff
@@ -42,17 +41,29 @@ describe('Admin - Employment Status API ', function () {
 
   describe('With `emp-statuses` snapshot', function () {
     before(function () {
-      cy.task('db:reset');
+      cy.task('db:truncate', {tables: ['EmploymentStatus']});
```

Comment:
can you revert these to `db:reset` as we discuss. will improve it in a separate PR

Reply:
reverted